app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/PreferencesFragment.java

```diff
@@ -62,14 +90,67 @@ public boolean onPreferenceClick(Preference preference) {
                 Intent contactIntent = new Intent(Intent.ACTION_SENDTO);
                 contactIntent.setData(Uri.parse(getString(R.string.off_mail)));
                 contactIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                try{
+                try {
                     startActivity(contactIntent);
-                }catch (android.content.ActivityNotFoundException e){
+                } catch (android.content.ActivityNotFoundException e) {
 
-                    Toast.makeText(getActivity(), R.string.email_not_found,Toast.LENGTH_SHORT).show();
+                    Toast.makeText(getActivity(), R.string.email_not_found, Toast.LENGTH_SHORT).show();
                 }
                 return true;
             }
         });
     }
+
+    private class GetAdditives extends AsyncTask<Void, Integer, Boolean> {
+
+        private static final String ADDITIVE_IMPORT = "ADDITIVE_IMPORT";
+        private LoadToast lt = new LoadToast(getActivity());
+
+
+        @Override
+        protected void onPreExecute() {
+            super.onPreExecute();
+            lt.setText(getActivity().getString(R.string.toast_retrieving));
+            lt.setBackgroundColor(ContextCompat.getColor(getActivity(), R.color.blue));
+            lt.setTextColor(ContextCompat.getColor(getActivity(), R.color.white));
+            lt.show();
+        }
+
+        @Override
+        protected Boolean doInBackground(Void... arg0) {
+            boolean result = true;
+
+            String language = LocaleHelper.getLanguage(getActivity());
+            String additivesFile = "additives_en.json";
+            additivesFile.replace("en", language);
```

Comment:
Why not replace with these three lines with one?
`String additivesFile = "additives_" + LocaleHelper.getLanguage(getActivity()) + ".json";`